supervisor/shared/filesystem.c

```diff
@@ -74,7 +74,7 @@ static void make_sample_code_file(FATFS *fatfs) {
     #if CIRCUITPY_FULL_BUILD
     FIL fs;
     UINT char_written = 0;
-    const byte buffer[] = "print('Hello World!')\n";
+    const byte buffer[] = "print("Hello World!")\n";
```

Comment:
You have to escape the inside double quotes:

```suggestion
    const byte buffer[] = "print(\"Hello World!\")\n";
```

Reply:
Ah! Ok! Done!